templates/social_partials/facebook_partial.html

```diff
@@ -0,0 +1,3 @@
+<div>
```

Comment:
Why does each social need it's own partial? They all have the same structure. Why not just an abstract `{{ connection.profile_url }}` and the connection handles generating it?